model/model.go

```diff
@@ -31,18 +33,43 @@ func newAppConfig() *AppConfig {
 	return &AppConfig{}
 }
 
+// BuilderConfig encapsulates the configuration of the deis-builder-- if it's in use.
+type BuilderConfig struct {
+	ConnectTimeout int `json:"connectTimeout"`
+	TCPTimeout     int `json:"tcpTimeout"`
+	ServiceIP      string
+}
+
+func newBuilderConfig() *BuilderConfig {
+	return &BuilderConfig{
+		ConnectTimeout: 10000,
+		TCPTimeout:     1200000,
+	}
+}
+
 // Build creates a RouterConfig configuration object by querying the k8s API for
 // relevant metadata concerning itself and all routable services.
 func Build(kubeClient *client.Client) (*RouterConfig, error) {
-	rc, err := getRC(kubeClient)
+	// Get all relevant information from k8s:
+	//   deis-router rc
+	//   All services with label "routable=true"
```

Comment:
this label is new to me. I'm assuming we'll need to add it to all applicable services (`deis-builder`, `deis-workflow`, etc...)? also, does that include user apps?